### DISCUSSION TITLE
Custom media queries for screen breakpoints? (like height)

### DISCUSSION THREAD

**Discussion by joseph-ampfer** [jaredh159/tailwind-react-native-classnames#302] [Q&A]
Is there support for adding breakpoints that are based on height like this from tailwind `'tall': { 'raw': '(min-height: 800px)' },`?  I noticed that this won't work for me, but I can adjust the min width fine like you did in the docs.

**Comment by jaredh159** [comment]
that wouldn't work in this library because it's only targeting native environments where there are no media queries. the current implementation for viewport width based prefixes is implemented based on the window width of the device, because we're working in a non-web environment. 

that doesn't meant that height-based queries can't work, it just means they can't be expressed in terms of raw web media queries.

**Reply by joseph-ampfer** [reply]
okay, thanks. ended up making breakpoints like  `export const med = 'min-h-[777px]:min-w-[392px]'` and seems to work fine
